PhotoViewer/image_handler.py

```diff
@@ -1,15 +1,12 @@
 from directories_handler import DirectoriesHandler
 from PIL import Image, ImageTk
-from tkinter import Label
 
 
 #$ Can you find a better name for self.Label and self.Image? so I can know what does they represent without reading all the class?
 class ImageHandler(DirectoriesHandler):
```

Comment:
You injected the directory handler here well. Why didnt you do the same with the image handler before ;)?